examples/platforms/efr32mg13/README.md

```diff
@@ -0,0 +1,266 @@
+[SLTB004A]: https://www.silabs.com/products/development-tools/thunderboard/thunderboard-sense-two-kit
+
+The example platform drivers are intended to present the minimal code
+necessary to support OpenThread. [EFR32MG13P SoC][efr32mg13p]
+has rich memory and peripheral resources which can support all OpenThread
+capabilities. See the "Run the example with EFR32MG13 boards" section below
+for an example using basic OpenThread capabilities.
+
+See [sleepy-demo/README.md](sleepy-demo/README.md) for instructions for an example that uses the low-energy
+modes of the EFR32MG13 when running as a Sleepy End Device.
+
+[efr32mg13p]: http://www.silabs.com/products/wireless/mesh-networking/efr32mg-mighty-gecko-zigbee-thread-soc/device.EFR32MG13P432F1024GL125
+
+## Toolchain
+
+Download and install the [GNU toolchain for ARM Cortex-M][gnu-toolchain].
+
+[gnu-toolchain]: https://developer.arm.com/open-source/gnu-toolchain/gnu-rm
+
+In a Bash terminal, follow these instructions to install the GNU toolchain and
+other dependencies.
+
+```bash
+$ cd <path-to-openthread>
+$ ./script/bootstrap
+```
+
+## Build Examples
+
+1. Download and install the [Simplicity Studio][simplicity_studio].
+
+[simplicity_studio]: http://www.silabs.com/products/development-tools/software/simplicity-studio
+
+2. Install Flex (Gecko) SDK including RAIL Library from Simplicity Studio.
+   - Connect EFR32MG13P Wireless Starter Kit to Simplicity Studio.
+   - Find Flex SDK v2.7 in the Software Update page and click Install.
+   - Flex SDK v2.7 will be installed in the path: `/SimplicityStudio_v4/developer/sdks/gecko_sdk_suite`.
+
+For more information on configuring, building, and installing applications for the Wireless Gecko (EFR32)
+portfolio using FLEX, see [Getting Started with the Silicon Labs Flex Software Development Kit for the 
+Wireless Gecko (EFR32™) Portfolio][QSG138]. For more information
+on RAIL, see [Radio Abstraction Interface Layer][rail].
+
+[QSG138]: https://www.silabs.com/documents/public/quick-start-guides/qsg138-flex-efr32.pdf
+[rail]: http://www.silabs.com/products/development-tools/software/radio-abstraction-interface-layer-sdk
+
+3. Configure the path to Flex SDK source code.
+```bash
+$ cd <path-to-openthread>/third_party
+$ mkdir silabs
+$ cd <path-to-Simplicity-Studio>/developer/sdks
+$ cp -rf gecko_sdk_suite <path-to-openthread>/third_party/silabs/
+```
+
+Alternatively create a symbolic link to the Flex SDK source code.
+```bash
+$ cd <path-to-openthread>/third_party
+$ mkdir silabs
+$ ln -s <path-to-Simplicity-Studio>/developer/sdks/gecko_sdk_suite silabs/gecko_sdk_suite
+```
+
+4. Build OpenThread Firmware (CLI example) on EFR32 platform.
+```bash
+$ cd <path-to-openthread>
+$ ./bootstrap
+```
+For EFR32MG13™ Mighty Gecko Wireless Starter Kit:
+```bash
+$ make -f examples/Makefile-efr32mg13 BOARD=BRD4161A
+```
+or alternatively for the Thunderboard™ Sense 2:
+
+```bash
+$ make -f examples/Makefile-efr32mg13 BOARD=BRD4166A
```

Comment:
BRD4166A does not seem to be supported for efr32mg13. Is this example for Thunderboard Sense 2 valid for the efr32mg13?

Reply:
Thunderboard Sense is of type mg12, so should not be mentoined in mg13 readme.